pkg/catalog/search.go

```diff
@@ -7,13 +7,16 @@ import (
 	"net/url"
 )
 
+// HTTPClient defines an http client which then can be used to test the
+// handler code.
 //go:generate counterfeiter -o fakes/fake_http_client.go . HTTPClient
 type HTTPClient interface {
 	Do(*http.Request) (*http.Response, error)
 }
 
 var httpClient HTTPClient = http.DefaultClient
 
+// Search will return profile descriptions for a given `catalogURL` and `profileName`.
 func Search(catalogURL, profileName string) ([]ProfileDescription, error) {
```

Comment:
How about `"Search queries the catalog at catalogURL for profiles matching the provided profileName."` ?

(Personally, I think `profileName` isn't quite right here, because that implies that the client knows how the server works, but that's probably not an issue right now).

Reply:
Yeah that sounds good to me.

Reply:
I have changed this to be something like `query` in my upcoming PR 👍

Reply:
Even better! :)